config/overrides.go

```diff
@@ -440,3 +440,16 @@ func KnownReferencers() tjconfig.ResourceOption { //nolint:gocyclo
 		}
 	}
 }
+
+// AddExternalTagsField adds ExternalTagsFieldName configuration for resources that have tags field.
+func AddExternalTagsField() tjconfig.ResourceOption {
+	return func(r *tjconfig.Resource) {
+		if s, ok := r.TerraformResource.Schema["tags"]; ok {
+			if s.Type == schema.TypeMap {
+				r.InitializerFns = []tjconfig.NewInitializerFn{
+					tjconfig.TagInitializer,
+				}
```

Comment:
This needs to be an `append` operation so that it doesn't override the existing ones that might have been set by other configurators.

Reply:
Done!